packages/binding-file/README.md

```diff
@@ -92,7 +92,7 @@ servient.addClientFactory(new FileClientFactory(null));
 
 let wotHelper = new Helpers(servient);
 wotHelper
-    .fetch("file://TD.jsonld")
+    .fetch("file:///TD.jsonld")
```

Comment:
Should we change this example using the new official `requestThingDescription` function?

Reply:
Good point, I would probably add that to #1166, though, alongside the rest of the file-related implementation of the new method.

Reply:
Ah, okay, nevermind, as #1166 got already merged ;) Then I will add it to this PR instead :)

Reply:
I added an initial update of the example in 1b79e91ea7696b86429f028fa0a0db84fdd88e73, haven't tested it yet, though.

Reply:
I now performed some tests and actually found a few bugs which I fixed in 3793b6ecbb76883e91dd2bcf25dee88d87cea1c5.